lib/spoom/cli/coverage.rb

```diff
@@ -44,6 +44,7 @@ def timeline
         path = exec_path
         sorbet = options[:sorbet]
 
+        branch_before = Spoom::Git.current_branch
         sha_before = Spoom::Git.last_commit(path: path)
```

Comment:
Can we merge the two?

```suggestion
        ref_before = Spoom::Git.current_branch
        ref_before = Spoom::Git.last_commit(path: path) unless ref_before
```

Then later:

```rb
Spoom::Git.checkout(ref_before, path: path)
```